concourse/scripts/publish_docs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -eou
+
+cd ./fauna-python-repository
+
+pip install -r requirements.txt
+
+PACKAGE_VERSION=$(python setup.py --version)
+
+pip install .
+pip install requests
+pip install pdoc3
+pdoc fauna --html -o docs
+
+cd ../
+mkdir docs
+cp -R ./fauna-python-repository/docs/fauna/* ./docs/
+
+echo "Current docs version: $PACKAGE_VERSION"
+
+apk add --no-progress --no-cache sed
+
+echo "================================="
+echo "Adding google manager tag to head"
+echo "================================="
+
+HEAD_GTM=$(cat ./fauna-python-repository/concourse/scripts/head_gtm.dat)
+sed -i.bak "0,/<\/title>/{s/<\/title>/<\/title>${HEAD_GTM}/}" ./docs/index.html
+
+echo "================================="
+echo "Adding google manager tag to body"
+echo "================================="
+
+BODY_GTM=$(cat ./fauna-python-repository/concourse/scripts/body_gtm.dat)
+sed -i.bak "0,/<body>/{s/<body>/<body>${BODY_GTM}/}" ./docs/index.html
+
+rm ./docs/index.html.bak
```

Comment:
Can you add a little more comment detail to this section?

e.g.
* Why we are making a `.bak` and then deleting it
* Why the tag goes after title but inside body (maybe just a link to docs about this)

Reply:
this is a copy and paste job 😁 -- https://github.com/fauna/faunadb-python/blob/v4/concourse/scripts/publish_docs.sh